doc/manual/chapters/translation.tex

```diff
@@ -1,10 +1,364 @@
+x[exp1]
+\* or
+x[exp1, exp2, exp3]
+\* or
+x[<<exp1, exp2, exp3>>]
+\* or
+x[[field1 |-> e1, field2 |-> e2]]
+	\end{lstlisting} &
+	Supported; compiled code dependent on the type of \lstinline[language=pcal]|x| (the function) \\
+	\hline
+	Binary operator call & \lstinline[language=pcal]|x /\ y = z + 1| & Supported; compiled as expected \\
+	\hline
+	Function (as struct literal) &
+	\begin{lstlisting}[language=pcal]
+[field1 \in exp1 |-> exp2,
+ field2 |-> exp3]
+	\end{lstlisting} &
+	Supported; compiled as sorted slice of key-value structs \\
+	\hline
+	Function set (as function literal) &
+	\begin{lstlisting}[language=pcal]
+[Nat -> Nat]
+\* or
+[Nat -> 1..3]
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	Function set (as struct) & \lstinline[language=pcal]|[1..3 -> 1..3]| & Unsupported \\
+	\hline
+	Function substitution &
+	\begin{lstlisting}[language=pcal]
+[f EXCEPT ![exp1] = exp2]
+\* or
+[f EXCEPT !.field = exp]
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	If expression &
+	\begin{lstlisting}[language=pcal]
+if condition
+   then thenExp
+   else elseExp
+	\end{lstlisting} &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+var result type;
+if condition {
+	result = thenExp
+} else {
+	result = elseExp
+}
+// result is used in place of
+// the expression hereafter
+	\end{lstlisting} \\
+	\hline
+	Tuple (as slice) & \lstinline[language=pcal]|<<exp1, exp2, exp3>>| & Compiled as slice when all its contents are of the same type
+	\begin{lstlisting}[language=golang]
+[]type{exp1, exp2, exp3}
+	\end{lstlisting} \\
+	\hline
+	Case expression &
+	\begin{lstlisting}[language=pcal]
+CASE x -> y
+  [] z -> p
+  [] OTHER -> other
+	\end{lstlisting} &
+	Compiled as
+	\begin{lstlisting}[language=golang]
+var result type;
+if x {
+	result = y
+	goto matched
+}
+if z {
+	result = p
+	goto matched
+}
+result = other
+matched:
+// result is used in place of
+// the expression hereafter
+	\end{lstlisting} \\
+	\hline
+	Existental &
+	\begin{lstlisting}[language=pcal]
+\E a, b, c : exp
+\* or
+\EE a, b, c : exp
+	\end{lstlisting} &
+	Unsupportable; TLC chokes when given this expression \\
+	\hline
+	Universal &
+	\begin{lstlisting}[language=pcal]
+\A a, b, c : exp
+\* or
+\AA a, b, c : exp
+	\end{lstlisting} &
+	Unsupportable; TLC chokes when given this expression \\
+	\hline
+	Let expression &
+	\begin{lstlisting}[language=pcal]
+LET op(a, b, c) == exp1
+    fn[d \in D] == exp2
+    e == exp3
+IN exp
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	Assumption &
+	\begin{lstlisting}[language=pcal]
+ASSUME exp
+\* or
+ASSUMPTION exp
+\* or
+AXIOM exp
+	\end{lstlisting} &
+	Unsupported \\
+	\hline
+	Theorem & \lstinline[language=pcal]|THEOREM exp| & Unsupported \\
+	\hline
+	Maybe action & \lstinline[language=pcal]|[exp1]_exp2| &
+	Unsupported \\
+	\hline
+	Required action & \lstinline[language=pcal]|<<exp1>>_exp2| & Unsupported \\
+	\hline
+	Operator call & \lstinline[language=pcal]|op(arg1, arg2)| & Supported; compiled as expected \\
```

Comment:
What is "as expected"? A function call? There's no entry for operators in this table.